src/components/Simulation.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { SimulatedTransaction, TransactionSimulation } from '../types/api';
+import { Collapsable } from './UI/Collapsable';
+
+import * as Styled from './Simulation.styles';
+
+interface SimulationProps {
+    simulation: TransactionSimulation;
+    defaultState?: boolean;
+}
+
+const shouldShowTransaction = (transaction: SimulatedTransaction) => {
+    return transaction.amount && transaction.symbol;
+}
+
+export function Simulation({ simulation, defaultState = false }: SimulationProps) {
+    const logo = <img src={'../assets/logo.png'} width='24' />;
```

Comment:
lets change the icon, something like this perhaps : https://iconscout.com/unicons/explore/line?redirect=%2Funicons%2Fexplore%2Fline%3Fcolor%3D%2525230092E4%26name%3Dexchange-alt